### PR TITLE
PMM-7585

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

File: pmm-app/tests/verifyPMMSettingsPageFunctionality_test.js [hunk 1]
```diff
@@ -136,7 +136,7 @@ Scenario('PMM-T520 - Verify that alert is being fired to external Alert Manager
   I.amOnPage(pmmSettingsPage.stateOfAlertsUrl);
   await pmmSettingsPage.verifyAlertmanagerRuleAdded(pmmSettingsPage.alertManager.ruleName2, true);
   await pmmSettingsPage.verifyExternalAlertManager(pmmSettingsPage.alertManager.ruleName2);
-}).retry(1);
+}).retry(0);
```
Comment (puneet0191) [inline pmm-app/tests/verifyPMMSettingsPageFunctionality_test.js:139]: Does not make sense to have a retry 0

**Review by puneet0191 — CHANGES_REQUESTED** [review]
Please pull changes from upstream PMM-2.0, upstream release/2.16 also address the comment left for retry

**Review by yurkovychv — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -61,6 +61,16 @@ pipeline {
                             sudo docker-compose --version
                         fi
                     '''
+                    sh '''
+                        sudo mkdir -p /srv/pmm-qa || :
+                        pushd /srv/pmm-qa
+                            sudo git clone --single-branch --branch PMM-7482 https://github.com/percona/pmm-qa.git .
```
Comment (puneet0191) [inline Jenkinsfile:67]: change the branch to master, the chrome script has been merged already

**Review by puneet0191 — CHANGES_REQUESTED** [review]

**Review by puneet0191 — APPROVED** [review]

File: pmm-app/package.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
     "lint:dev": "eslint --ext .tsx,.ts --fix src/",
     "lint:tests": "eslint --ext .js --fix tests/",
     "jest": "jest",
-    "e2e": "docker run --net=host -v $PWD:/tests -v $PWD/node_modules:/node_modules --env-file env.generated.list codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
+    "e2e": "./node_modules/.bin/codeceptjs run-multiple parallel --debug --steps --reporter mocha-multi -c pr.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
     "e2e:local": "docker run --net=host --name codecept -v $PWD:/tests -v $PWD/node_modules:/node_modules codeception/codeceptjs:2.6.1 codeceptjs run-multiple parallel --all --debug --steps -c pr-local.codecept.js --grep '(?=.*)^(?!.*@not-ui-pipeline)^(?!.*@not-pr-pipeline)'",
```
Comment (puneet0191) [inline pmm-app/package.json:24]: @IvanaBrecherova change this also please, we can not use docker based runs any more

**Review by puneet0191 — CHANGES_REQUESTED** [review]

**Pull request merged**